paper/paper.md

```diff
@@ -52,25 +52,17 @@ performing numerical orbit integration).
 
 # Statement of need
 
-`Gala` is an Astropy-affiliated Python package for galactic dynamics. Python
-enables wrapping low-level languages (e.g., C) for speed without losing
-flexibility or ease-of-use in the user-interface. The API for `Gala` was
-designed to provide a class-based and user-friendly interface to fast (C or
-Cython-optimized) implementations of common operations such as gravitational
-potential and force evaluation, orbit integration, dynamical transformations,
-and chaos indicators for nonlinear dynamics. `Gala` also relies heavily on and
-interfaces well with the implementations of physical units and astronomical
-coordinate systems in the `Astropy` package [@astropy] (`astropy.units` and
-`astropy.coordinates`).
-
-`Gala` was designed to be used by both astronomical researchers and by
-students in courses on gravitational dynamics or astronomy. It has already been
-used in a number of scientific publications [@Pearson:2017] and has also been
-used in graduate courses on Galactic dynamics to, e.g., provide interactive
-visualizations of textbook material [@Binney:2008]. The combination of speed,
-design, and support for Astropy functionality in `Gala` will enable exciting
-scientific explorations of forthcoming data releases from the *Gaia* mission
-[@gaia] by students and experts alike.
+`mesa-frames` is an extension to the `mesa` [@python-mesa-2020] agent-based modeling framework in Python, designed to enhance performance and scalability for complex simulations involving millions of agents.
+
+`mesa` has become the most widely used framework for agent-based modeling in Python thanks to its easy-to-use API and object-oriented philosophy. However, iterating through each agent's behavior becomes computationally expensive when the number of agents reaches thousands [@app13010013]. This has led to the development of other frameworks like `Agents.jl` [@Agents.jl], which requires developing in another language, Julia. mesa-frames aims to achieve performance similar to that of `Agents.jl` (NOTE: this needs to be proved) but with a simpler Python syntax.
```

Comment:
I checked https://arxiv.org/pdf/2101.10072, and there was no evidence that Agents.jl was a reaction to Mesa being slow. Additionally, in the paper:
> From our perspective, the biggest take-away of this paper is that Agents.jl is a framework that is simple to use, requiring small amount of written code from the user and overall easy to learn. Despite this, our comparison shows that Agents.jl always exceeds other frameworks in performance, and often also in capability

Reply:
You're right there's no consequence of Agents.jl being born out of mesa's speed. I just wanted to remark that the performance of mesa-frames is similar to the one of Agents.jl but the code is in Python. 
Maybe it's better to leave out Agents.jl completely?

Reply:
Concrete benchmark comparison with Agents.jl is still worth it. They have the [official benchmark repo](https://github.com/JuliaDynamics/ABMFrameworksComparison).

Reply:
This is very useful. Before implementing the models needed for the benchmark, I would reimplement mesa-frames in Ibis.